app/app/v1alpha/app_private_service.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package app.app.v1alpha;
+
+import "app/app/v1alpha/app.proto";
+
+// AppPrivateService exposes the private endpoints that allow clients to
+// manage apps.
+service AppPrivateService {
+  // List the tags in a repository.
+  //
+  // Returns a portion of the versions that the specified repository holds.
+  rpc ListRepositoryTags(ListRepositoryTagsRequest) returns (ListRepositoryTagsResponse);
+
+  // Get details of repository tag.
+  rpc GetRepositoryTag(GetRepositoryTagRequest) returns (GetRepositoryTagResponse);
+
+  // Create a new repository tag.
+  //
+  // Adds a tag to a given repository. Note that this operation is only
+  // intended to register the information of an *already created* tag. This
+  // method should be called as part of the content push operation, right after
+  // the [PUT Manifest](https://distribution.github.io/distribution/#put-manifest) has
+  // succeeded. The distribution registry won't hold data such as the push time
+  // or the tag digest, so `app-backend` will hold this information locally.
+  rpc CreateRepositoryTag(CreateRepositoryTagRequest) returns (CreateRepositoryTagResponse);
+
+  // Delete a repository tag.
+  rpc DeleteRepositoryTag(DeleteRepositoryTagRequest) returns (DeleteRepositoryTagResponse);
```

Comment:
I think we don't need these, its for docker image operation in artifact-backend

Reply:
right. I have removed that part. please check again. 🙇‍♂️